### ISSUE TITLE
Change role of clickable series svg

### ISSUE THREAD

**Issue by mauro1855** [highcharts/highcharts#19751]
# Description of the feature
It is currently possible in Highcharts to define click events for points in a series (documentation [here](https://api.highcharts.com/highcharts/plotOptions.series.point.events.click)) as well as define a specific cursor for the series  (documentation [here](https://api.highcharts.com/highcharts/plotOptions.series.cursor)). With these two features, the series will act and look like a button, but for A11Y purposes it is not a button.
Our accessibility team believes that, when using these features, the svg graphic should have a role of 'button' so that assistive technologies (like Voice Over) properly announce the series as clickable.

Feature request: allow defining a series' graphic HTML role attribute as 'button'

# Library related to the feature
Highcharts Accessibility (?)

**You can vote for this feature by adding a thumbs-up reaction to this post.**

**Comment by mauro1855** [comment]
Note, I think this is a similar (if not the same) as point 3. in https://github.com/highcharts/highcharts/issues/17841

**Comment by oysteinmoseng** [comment]
Thanks for bringing this up! Is your use case about drilldown, or do you have a more generic use for the series click?
